src/api/api.js

```diff
@@ -0,0 +1,19 @@
+import * as axios from "axios";
+
+const instance = axios.create({
+    withCredentials: true,
+    baseURL: 'https://react-mipt-course-server.herokuapp.com/'
```

Comment:
You will have a CORS problem with this approach. For testing, I suggest changing this to '/':
```
const instance = axios.create({
    withCredentials: true,
    baseURL: '/'
});
```
And add `"proxy": "https://react-mipt-course-server.herokuapp.com/"` to `package.json`

Reply:
And look at the error you will get from the network. The server expects field `email` instead of `login` for example)